Task3_Position_the_Cubes/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div class="Parent">1
+        <div class="child1">2</div>
+        <div class="child2">2</div>
+        <div class="child3">2</div>

```

Comment:
The text content '2' should be inside the closing tag of the child3 div, like this: <div class='child3'>2</div>